testing/test_integration.py

```diff
@@ -377,6 +377,21 @@ def test_function(arg):
         assert_that(col_name).contains("::setup")
         assert_that(get_log(page)).contains("ValueError")
 
+    def test_chdir(self, pytester):
```

Comment:
Once this fails without the fix, it should probably live in `test_unit.py` as it really doesn't assert anything in the generated report.

Reply:
Indeed I moved it.
Now it fails for me without the fix.